backend/plugins/frontline_api/src/modules/tickets/utils.ts

```
@@ -0,0 +1,1060 @@
+  const resolver = resolvers['Ticket'] || {};
+
+  for (const subResolver of Object.keys(resolver)) {
+    try {
+      additionInfo[subResolver] = await resolver[subResolver](
+        ticket,
+        {},
+        { models, subdomain, user },
+        { isSubscription: true },
+      );
+    } catch (unused) {
+      continue;
```

Biome (1.9.4)

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)